### PR TITLE
Enhanced mask summary

### PR DESCRIPTION
**Pull request by VasylTretiakov**
1. rephrased to mention uniform masks, possibility to combine drawn and parametric masks, possibility to reuse masks in several modules (which includes a weak reference to raster masks)
2. clarified that masks can be applied in blend modes

### CODE REVIEW

File: doc/usermanual/overview/overview.xml [hunk 1]
```diff
@@ -103,16 +103,16 @@
     </para></listitem>
 
     <listitem><para>
-      Many darktable modules can be combined with blending operators for even more development
-      options.
+      A powerful mask feature gives you fine control over extent the module affects complete image
+      or its specific parts. To select image areas you draw a mask using various shapes and/or
```
Comment (Balise42) [inline doc/usermanual/overview/overview.xml]: Matter of taste, I think: I'd tend to say "You can select image areas by drawing a mask using various shapes, or by defining a parametric mask based on pixel values."

File: doc/usermanual/overview/overview.xml [hunk 2]
```diff
@@ -103,16 +103,16 @@
     </para></listitem>
 
     <listitem><para>
-      Many darktable modules can be combined with blending operators for even more development
-      options.
+      A powerful mask feature gives you fine control over extent the module affects complete image
```
[the remark this answers is not in the document]
Comment (VasylTretiakov) [reply doc/usermanual/overview/overview.xml]: merged second option

File: doc/usermanual/overview/overview.xml (hunk 1 above)
Comment (VasylTretiakov) [reply doc/usermanual/overview/overview.xml]: merged

**Comment by TurboGit** [comment]
Thanks !

**Pull request merged**